modules/stanford_profile_styles/lib/scss/paragraph-types/react_paragraphs/_multirow.scss

```diff
@@ -0,0 +1,144 @@
+@charset 'UTF-8';
+
+// For full width pages
+.jumpstart-ui--one-column {
+  .react-paragraphs-row {
+
+    // Setting the max width to help with display and
+    // to keep the margins on full-width pages.
+    .paragraph-item {
+
+      // 4 items across a page.
+      &.flex-3-of-12 {
+        &.ptype-stanford-banner {
+          @include grid-media-between('xs', 'md') {
+            min-width: 100%
+          }
+        }
+
+        &.ptype-stanford-card,
+        &.ptype-stanford-wysiwyg,
+        &.ptype-stanford-media-caption {
+          @include grid-media('xl') {
+            @include margin(null $su-gutter-xl);
+            max-width: 20%;
+          }
+
+          @include grid-media-between('lg', 'xl') {
+            @include margin(null $su-gutter-lg);
+            max-width: 18%;
+          }
+        }
+      }
+
+      // Three items across a page.
+      &.flex-4-of-12 {
+        @include grid-media-between('lg','xl') {
+          @include margin(null 0);
+          max-width: 33%;
+        }
+
+        @include grid-media-between('xs', 'md') {
+          max-width: 100%;
+        }
+
+        &.ptype-stanford-banner {
+          @include grid-media-between('xs', 'md') {
+            min-width: 100%
+          }
+        }
+
+        &.ptype-stanford-card,
+        &.ptype-stanford-wysiwyg,
+        &.ptype-stanford-media-caption {
+          @include grid-media('xl') {
+            @include margin(null $su-gutter-xl);
+            max-width: 28%;
+          }
+
+          @include grid-media-between('lg', 'xl') {
+            @include margin(null $su-gutter-lg);
+            max-width: 26%;
+          }
+        }
+      }
+
+      // 2 items across a page.
+      &.flex-6-of-12 {
+        &.ptype-stanford-banner {
+          @include grid-media-between('xs', 'md') {
+            min-width: 100%
+          }
+        }
+        &.ptype-stanford-card,
+        &.ptype-stanford-wysiwyg,
+        &.ptype-stanford-media-caption {
+          @include grid-media('xl') {
+            @include margin(null $su-gutter-xl);
+            max-width: 45%;
+          }
+          @include grid-media-between('lg','xl') {
+            @include margin(null $su-gutter-lg);
+            max-width: 43%;
+          }
+        }
+
+        @include grid-media('lg') {
+          @include margin(null 0);
+          max-width: 50%;
+        }
+
+        @include grid-media-between('xs', 'md') {
```

Comment:
This can be `grid-media-max`

Reply:
I can see how this is better since there is only one variable, the 'md', so it is easier to read?

Reply:
The 'xs' breakpoint is equal to 0 so there is nothing before it. This is a micro-optimization and in the grand scheme of things, it more helping us as developers be able to read it and prevent less hilarity if we change xs to something else later.